packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -188,9 +189,9 @@ describe('KeyringController', () => {
           await withController(
             { cacheEncryptionKey },
             async ({ controller, initialState }) => {
-              const currentSeedWord = await controller.exportSeedPhrase(
+              const currentSeedWord = (await controller.exportSeedPhrase(
```

Comment:
What's the reason for typecasting this? If this is due to a deficiency what do you think about adding a comment explaining why?

Reply:
Indeed this is not needed anymore!